src/ontology/components/mappings.owl

```diff
@@ -159,326 +159,340 @@
         <sssom:mapping_justification rdf:resource="https://w3id.org/semapv/vocab/ManualMappingCuration"/>
     </owl:Axiom>
     <rdf:Description rdf:about="http://purl.obolibrary.org/obo/UBERON_0000066">
+        <oboInOwl:hasDbXref>BtauDv:0000078</oboInOwl:hasDbXref>
+        <oboInOwl:hasDbXref>CfamDv:0000007</oboInOwl:hasDbXref>
+        <oboInOwl:hasDbXref>ChirDv:0000014</oboInOwl:hasDbXref>
+        <oboInOwl:hasDbXref>DpseDv:0000005</oboInOwl:hasDbXref>
+        <oboInOwl:hasDbXref>DsimDv:0000005</oboInOwl:hasDbXref>
+        <oboInOwl:hasDbXref>EcabDv:0000007</oboInOwl:hasDbXref>
+        <oboInOwl:hasDbXref>FcatDv:0000007</oboInOwl:hasDbXref>
+        <oboInOwl:hasDbXref>GgalDv:0000076</oboInOwl:hasDbXref>
+        <oboInOwl:hasDbXref>GgorDv:0000032</oboInOwl:hasDbXref>
         <oboInOwl:hasDbXref>HsapDv:0010000</oboInOwl:hasDbXref>
+        <oboInOwl:hasDbXref>MmulDv:0000052</oboInOwl:hasDbXref>
         <oboInOwl:hasDbXref>MmusDv:0000092</oboInOwl:hasDbXref>
-        <oboInOwl:hasDbXref>obo:BtauDv_0000078</oboInOwl:hasDbXref>
```

Comment:
Normal consequences of a fix that happened in the SSLSO ontology in the past few months.